src/08_comprehensions.py

```diff
@@ -2,22 +2,22 @@
 List comprehensions are one cool and unique feature of Python.
```

Comment:
Perfect! 🎉  List comprehensions and slice notations will come in handy a lot throughout CS.